### PR TITLE
Add in an app usage page

### PR DESCRIPTION
**Pull request by andymckay**
* re-use credits style, but since its for more than credits rename it
* add in usage page
* add in web activity

Not sure on unit tests, any advice?

r? @diox also fyi: @marshall

### CODE REVIEW

File: src/media/css/standalone.styl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.standalone {
+    ul {
+        list-style: none;
+    }
+    li {
+        padding: 3px 0;
+    }
+    h3 {
+        text-transform: uppercase;
```
Comment (ngokevin) [inline src/media/css/standalone.styl]: try ```_h3();```

File: src/dev.html [hunk 2]
```diff
@@ -71,6 +70,7 @@
       <link rel="stylesheet" href="/media/css/reviews.styl.css">
       <link rel="stylesheet" href="/media/css/search.styl.css">
       <link rel="stylesheet" href="/media/css/settings-hover-desktop.styl.css">
+      <link rel="stylesheet" href="/media/css/standalone.styl.css">
```
Comment (ngokevin) [inline src/dev.html]: not sure what standalone means. Maybe we should just pull the styles into our base CSS.

File: src/media/js/views/usage.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+define('views/usage',
+    ['l10n', 'log', 'models', 'urls', 'z'],
+    function(l10n, log, models, urls, z) {
+    'use strict';
+
+    var gettext = l10n.gettext;
+
+    return function(builder, args, params) {
+        builder.start('usage.html');
+        builder.z('type', 'leaf');
```
Comment (ngokevin) [inline src/media/js/views/usage.js]: nit: i like having builder.z on top of builder.start

File: src/dev.html (hunk 2 above)
Comment (andymckay) [inline src/dev.html]: Not a listings page, but a random page. Dunno, any advice on naming welcome.

File: src/media/js/views/usage.js (hunk 3 above)
Comment (andymckay) [inline src/media/js/views/usage.js]: k

File: src/templates/usage.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <h1>
+      {{ _('App Statistics') }}
+    </h1>
+  </div>
+</section>
+
+<section class="standalone main full">
+  <div>
+    <h3>App usage data</h3>
+    <p>Not enough data to show.</p>
```
Comment (andymckay) [inline src/templates/usage.html]: Adding in some l10n here.

**Pull request closed without merging**